### PR TITLE
completed logout logic

### PR DESCRIPTION
**Pull request by Tanvir04khan**

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -19,10 +19,7 @@ const HomeScreenLayout = ({ children }) => {
     <>
       <Header onToggleSidebar={ToggleSidebarHandler} />
       <div className="app__container">
-        <SideBar
-          isSidebarVisible={isSidebarVisible}
-          onToggleSidebar={ToggleSidebarHandler}
```
Comment (faran4engg) [inline src/App.jsx:24]: function name. I approved with an assumption that you will rename it correctly and then merge. I see wrong update pf the name here @Tanvir04khan

File: src/redux/action/auth.action.js [hunk 2]
```diff
@@ -37,3 +37,17 @@ export const login = () => async (dispatch) => {
     });
   }
 };
+
+export const logout = () => async (dispatch) => {
+  await auth.signOut();
+
+  dispatch({
+    type: actions.LOG_OUT,
```
Comment (faran4engg) [inline src/redux/action/auth.action.js]: `actionTypes`

**Review by faran4engg — CHANGES_REQUESTED** [review]
request changes

File: src/App.jsx (hunk 1 above)
Comment (Tanvir04khan) [reply src/App.jsx:24]: changed the function's name

File: src/redux/action/auth.action.js (hunk 2 above)
Comment (Tanvir04khan) [reply src/redux/action/auth.action.js]: changed code as per suggestion

**Pull request merged**